packages/back-end/src/models/MetricModel.ts

```diff
@@ -108,6 +108,13 @@ export async function insertMetric(metric: Partial<MetricInterface>) {
   return toInterface(await MetricModel.create(metric));
 }
 
+export async function insertMetrics(metrics: Partial<MetricInterface>[]) {
```

Comment:
Updated to use `Pick` - the `MetricInterface` is very verbose, so instead of using the full interface (which I don't think is possible in this situation, I opted for `Pick` so we can be explicit about exactly what we should be passing into this method.